web/client/translations/data.fr-FR.json

```diff
@@ -1502,6 +1502,10 @@
                 "label": "Définir la limite de visibilité",
                 "tooltip": "Applique automatiquement les limites de visibilité suggérées par le serveur"
             },
+            "singleTile": {
+                "label": "Single Tile",
+                "tooltip": "La couche est rendue sous la forme d'une seule image de tuile lorsqu'elle est ajoutée à la carte avec cette option activée"
```

Comment:
*sous la forme d'une seule image lorsqu'elle* will be clearer i think.

Reply:
@landryb like this
```suggestion
                "tooltip": "La couche est rendue sous la forme d'une seule image lorsqu'elle est ajoutée à la carte avec cette option activée"
```

Reply:
yes i think so. @MaelREBOUX do you think that's meaningful enough ?

or rather *Lorsque cette option est activée, la couche ajoutée à la carte est rendue sous la forme d'une seule image* ?